sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Hotspots/DoNotHardcodeCredentialsBase.cs

```diff
@@ -155,6 +158,17 @@ private IEnumerable<string> FindCredentialWords(string variableName, string vari
                 // before reporting to avoid new issues to appear on SQ/SC.
                 return credentialWordsFound.Select(x => x.ToLowerInvariant());
             }
+
+            private bool IsValidParameter(string suffix)
+            {
+                var candidateParameter = suffix
```

Comment:
_nitpicking_: Commands are super-short in this case. I would put them on single line.